docs/protocol/extension_logging.md

```diff
@@ -196,3 +212,74 @@ message LogSettingsResponse
   map<string, SettingValue> settings = 1;
 }
 ```
+
+## Logging Formats
+
+The logging extension offers two logging formats. The formats have a
+common set of fields but differ in how the timestamp for a log entry
+is represented. Messages are serialized according to JSON encoding
+rules by default. This behavior can be disabled by setting the
+environment variable TRITON_SERVER_ESCAPE_LOG_MESSAGES to "0" when
+launching the server but can not be changed through the logging
+extension.
+
+Log entries can be single-line or multi-line. Multi-line entries have
+a single optional heading followed by the structured representation of
+an object such as a table or protobuf message. Multi-line entries end
+when the next log entry begins.
+
+1. TRITONSERVER_LOG_DEFAULT
+
+### Single-line Entry
+```
+<level><month><day><hour>:<min>:<sec>.<usec> <pid> <file>:<line>] <message>
+```
+Example:
+```
+I0520 20:03:25.829575 3355 model_lifecycle.cc:441] "AsyncLoad() 'simple'"
+```
+### Multi-line Entry
+```
+<level><month><day><hour>:<min>:<sec>.<usec> <pid> <file>:<line>] <heading>
+<object>
+```
+Example:
+
+```
+I0520 20:03:25.912303 3355 server.cc:676]
++--------+---------+--------+
+| Model  | Version | Status |
++--------+---------+--------+
+| simple | 1       | READY  |
++--------+---------+--------+
+```
+
+
+2. TRITONSERVER_LOG_ISO8601
+
+### Single-line Entry
+```
+<year>-<month>-<day>T<hour>:<min>:<sec>Z <level> <pid> <file>:<line>] <message>
```

Comment:
same question as on core, does the format include `.<msec>`?

Reply:
it doesn't currently - that is a future enhancement though - we decided not to add it in these set of changes